test/spec/SearchParameters/isNumericRefined.js

```diff
@@ -1,59 +1,52 @@
 'use strict';
 
-var test = require('tape');
 var SearchParameters = require('../../../src/SearchParameters');
 
-test('isNumericRefined with 3 parameters', function(t) {
+test('isNumericRefined with 3 parameters', function() {
   var params = new SearchParameters();
 
-  t.notOk(params.isNumericRefined('age', '>', '3'));
-  t.notOk(params.isNumericRefined('age', '>', '7'));
-  t.notOk(params.isNumericRefined('age', '<', '7'));
-  t.notOk(params.isNumericRefined('size', '>', '3'));
+  expect(params.isNumericRefined('age', '>', '3')).toBeFalsy();
+  expect(params.isNumericRefined('age', '>', '7')).toBeFalsy();
+  expect(params.isNumericRefined('age', '<', '7')).toBeFalsy();
+  expect(params.isNumericRefined('size', '>', '3')).toBeFalsy();
 
   var paramsWithNumerics = (params.addNumericRefinement('age', '>', '3'));
 
-  t.ok(paramsWithNumerics.isNumericRefined('age', '>', '3'));
-  t.ok(paramsWithNumerics.isNumericRefined('age', '>', 3));
-  t.notOk(paramsWithNumerics.isNumericRefined('age', '>', '7'));
-  t.notOk(paramsWithNumerics.isNumericRefined('age', '<', '7'));
-  t.notOk(paramsWithNumerics.isNumericRefined('size', '>', '3'));
+  expect(paramsWithNumerics.isNumericRefined('age', '>', '3')).toBeTruthy();
```

Comment:
`isNumericRefined` returns a boolean so we can make the assertion more strict.

Reply:
https://github.com/algolia/algoliasearch-helper-js/pull/660#discussion_r278628453